src/common.rs

```diff
@@ -929,6 +933,14 @@ pub enum ProcessStatus {
     ///
     /// Not available.
     LockBlocked,
+    /// ## Linux
+    ///
+    /// Waiting in uninterruptible disk sleep.
+    ///
+    /// ## Other OS
+    ///
+    /// Not available.
+    UninterruptibleDiskSleep,
```

Comment:
FYI `man proc` calls this "Uninterruptible disk sleep" whereas `man ps` calls it "Uninterruptible sleep (usually IO)". I think going with "Uninterruptible Disk Sleep" makes more sense here, especially as `D` presumably used to stand for "Disk".

Reply:
I'll keep it as is then.